### DISCUSSION TITLE
use wp-i18n to translate strings.

### DISCUSSION THREAD

**Discussion by esradev** [gregsullivan/_tw#120] [General]
How I can use @wordpress/i18n package to make the theme strings used on javascript part ready for traslation and use wp-i18n command tool to generate .pot update .po and make .mo and .json files for that. 

I alreadey have setup my project and the wp-i18n commands will work with the bundled file that generated by wp-scripts, but when I use ESbuild to bundle js files, the wp-i18n commands not detect any string to translate.

**Comment by gregsullivan** [comment]
Hello—Thanks for posting this!

Off the top of my head, I think it could be an issue with esbuild needing a bit of additional work to recognize the `@wordpress/i18n` package. [This blog post](https://deluxeblogtips.com/esbuild/) has some thoughts as far as that goes.

Are you able to share your repository? It's difficult to say anything definitive without seeing how you've set up the JavaScript side of things.

**Comment by gregsullivan** [comment]
(I'm going to convert this to a discussion since it doesn't sound like an issue with _tw itself.)
